<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QMainWindow" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QToolButton" name="muteButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/volume.png</normaloff>
        <normalon>:/buttons/mute.png</normalon>:/buttons/volume.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="11">
     <widget class="QSlider" name="seekBar">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3">
     <widget class="QToolButton" name="shuffleButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/shuffle.png</normaloff>:/buttons/shuffle.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="14">
     <widget class="QLabel" name="currentSong">
      <property name="text">
       <string>No Playing Track</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QToolButton" name="playButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/play.png</normaloff>:/buttons/play.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="volumeBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="11">
     <widget class="QToolButton" name="removeButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/minus.png</normaloff>:/buttons/minus.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="11" colspan="3">
     <widget class="QLabel" name="timeStamp">
      <property name="text">
       <string>00:00 / 00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QToolButton" name="nextButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/skip-next.png</normaloff>:/buttons/skip-next.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QToolButton" name="prevButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/skip-previous.png</normaloff>:/buttons/skip-previous.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="13">
     <widget class="QToolButton" name="saveButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/save.png</normaloff>:/buttons/save.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QToolButton" name="repeatButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/repeat.png</normaloff>:/buttons/repeat.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="10">
     <widget class="QToolButton" name="addButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/plus.png</normaloff>:/buttons/plus.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="14">
     <widget class="QLineEdit" name="searchBar">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Search track...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="12">
     <widget class="QToolButton" name="clearButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="buttons.qrc">
        <normaloff>:/buttons/clear.png</normaloff>:/buttons/clear.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="14">
     <widget class="QListWidget" name="playlistView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSavePlaylist"/>
    <addaction name="actionClearQueue"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="aboutMenu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="aboutMenu"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/plus.png</normaloff>:/buttons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="actionSavePlaylist">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/save.png</normaloff>:/buttons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить плейлист</string>
   </property>
  </action>
  <action name="actionClearQueue">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/clear.png</normaloff>:/buttons/clear.png</iconset>
   </property>
   <property name="text">
    <string>Очистить очередь воспроизведения</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/exit.png</normaloff>:/buttons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="buttons.qrc">
     <normaloff>:/buttons/info.png</normaloff>:/buttons/info.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
